src/store/attachment.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * @copyright Copyright (c) 2018 Julius Härtl <jus@bitgrid.net>
```

Comment:
You created those files, make sure to add yourself to the copyright and author header :wink: